### PR TITLE
Feature/run storages

### PR DESCRIPTION
**Pull request by nguyeda1**
Run storages are available via `/v2/actor-runs/{runId}/{key-value-store | log | request-queue | dataset}` .. see https://docs.apify.com/api/v2#/reference/actor-runs/run-object-and-its-storages

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+1.2.5 / 2021/06/15
+===================
+- Enabled access to actor run storages via `RunClient`
```
Comment (mnmkng) [inline CHANGELOG.md:3]: Should be a minor version bump to `1.3.0`, since features were added.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "apify-client",
-    "version": "1.2.4",
+    "version": "1.2.5",
```
Comment (mnmkng) [inline package.json:3]: dtto

**Review by mnmkng — APPROVED** [review]
Ok, now I'm a bit torn. I thought you would need to add some code, but I scrolled down and it seems that you only added docs and the code was already there. So I guess you can ignore my comments about the version 1.3.0... 🤷‍♂️

**Review by drobnikj — CHANGES_REQUESTED** [review]
Please add simple tests.
There is a skeleton prepared already.
https://github.com/apify/apify-client-js/blob/master/test/runs.test.js#L204

**Comment by nguyeda1** [comment]
Added the same tests as for `actor().lastRun()`

**Comment by nguyeda1** [comment]
> Please add simple tests.
> There is a skeleton prepared already.
> https://github.com/apify/apify-client-js/blob/master/test/runs.test.js#L204

Added the same tests as for actor().lastRun()

**Review by drobnikj — APPROVED** [review]

**Pull request merged**